src/raftstore/store/peer.rs

```diff
@@ -248,7 +248,7 @@ impl Peer {
             pending_cmds: Default::default(),
             peer_cache: store.peer_cache(),
             coprocessor_host: CoprocessorHost::new(),
-            size_diff_hint: 0,
+            size_diff_hint: cfg.region_check_size_diff,
```

Comment:
scan all regions will definitly raise high IO load and last for a while when store contain many GB datas when start, and this make sense to user and system